src/store/selectors.ts

```diff
@@ -0,0 +1,32 @@
+import { reducerName as sessionReducerName } from '@onaio/session-reducer';
```

Comment:
Let us create an issue to move these into a package.

Reply:
link to issue: https://github.com/OpenSRP/web/issues/25